packages/elements/src/components.d.ts

```diff
@@ -775,13 +775,17 @@ export namespace Components {
     }
     interface InoNavItem {
         /**
-          * Styles the row in an activated style.  Use this for only one item and to mark it as permantently activated.
+          * Styles the row in an activated style.  Use this for only one item and to mark it as permanently activated.
          */
         "inoActivated"?: boolean;
         /**
           * Styles the row in a disabled style.
          */
         "inoDisabled"?: boolean;
+        /**
+          * The secondary text of this list item used in a two-lined list.
+         */
+        "inoSecondaryText"?: string;
```

Comment:
the term `Secondary` implies that there is also a `pirmaryText`. Because of this and according to `inoText` i would recommend renaming  the property to `inoSubText`.

Reply:
Ye, I was thinking about that too. For consistency reasons, we should rename `inoText` to `inoPrimaryText` but I did not want to introduce a breaking change for this yet.

I'll change it to `ino-sub-text` for now.